include/eve/literals.hpp

```diff
@@ -29,27 +29,88 @@ namespace eve::detail
 
 namespace eve::literal
 {
-  // integral constants
-  template<char... c> constexpr auto operator"" _c()
-  {
-    return detail::idx_<detail::parse<c...>()>{};
-  }
+  //================================================================================================
+  //! @addtogroup simd_types
+  //! @{
+  //! @defgroup literal Custom literal suffix
+  //! @brief  SIMD types related literal suffixes
+  //!
+  //! **EVE** provides various user-defined literal suffixes to simplify the definition of constants
+  //! in some of its API. Using them requires the following include and namespace usage:
+  //!
+  //! @code
+  //! using namespace eve::literal;
+  //! #include <eve/literals.hpp>
+  //! @endcode
+  //!
+  //! @groupheader{Index literals}
+  //! Some **EVE** function requires a constexpr constant as a parameter to identify a lane or an
+  //! an index in some of **EVE** components. The `_idx` suffix allows for a fast way to specify
+  //! such constant
+  //!
+  //! @code
+  //! using namespace eve::literal;
+  //! eve::wide<float, eve::fixed<4>> x = {0.1,0.01,0.001,0.0001};
+  //!
+  //! // Broadcast 1st element of x into y
+  //! auto y = eve::broadcast(x, 0_idx);
+  //! @endcode
+  //!
+  //! @groupheader{Wide literals}
+  //! Suffixes are provided to construct a eve::wide instance from a literal constant.
+  //!
+  //! @code
+  //! using namespace eve::literal;
+  //! // x is a eve::wide<short>
+  //! auto x = 45_wi16;
+  //!
+  //! // x is a eve::wide<float>
+  //! auto y = 2.635_wf;
```

Comment:
Omg rly? :)
Why would we need this?

Reply:
It's an old piece of code I forgot existed and I documented it.
I am not attached to it so if you want we can remove it.

Reply:
Let's remove them